apps/web/app/pages/sbtc-rewards/sbtc-rewards.tsx

```diff
@@ -49,68 +54,20 @@ const sbtcEnroll = {
   payoutToken: 'sBTC',
 } as const;
 
-// Fakes values, to be updated
-const pools = [
-  {
-    id: 'alex',
-    url: 'https://app.alexlab.co/pool',
-    logo: <AlexLogo size="32px" />,
-    title: 'ALEX',
-    description:
-      'Commit sBTC to liquidity pools to earn fees and LP tokens in addition to basic sBTC rewards',
-    tvl: '1,880 BTC',
-    tvlUsd: '$113,960,000',
-    minCommitment: '0.01 BTC',
-    minCommitmentUsd: '$605.00',
-    apr: '5.2%',
-    payoutToken: 'sBTC',
-  } as const,
-  {
-    id: 'bitflow',
-    url: 'https://app.bitflow.finance/sbtc#earn3',
-    logo: <BitflowLogo size="32px" />,
-    title: 'Bitflow',
-    description:
-      'Commit sBTC to liquidity pools to earn fees and LP tokens in addition to basic sBTC rewards',
-    tvl: '1,420 BTC',
-    tvlUsd: '$86,020,000',
-    minCommitment: '0.008 BTC',
-    minCommitmentUsd: '$484.00',
-    apr: '5.1%',
-    payoutToken: 'sBTC',
-  } as const,
-  {
-    id: 'velar',
-    logo: <VelarLogo size="32px" />,
-    url: 'https://app.velar.com/pool',
-    title: 'Velar',
-    description:
-      'Commit sBTC to liquidity pools to earn fees and LP tokens in addition to basic sBTC rewards',
-    tvl: '3,100 BTC',
-    tvlUsd: '$187,050,000',
-    minCommitment: '0.015 BTC',
-    minCommitmentUsd: '$907.50',
-    apr: '5.0%',
-    payoutToken: 'sBTC',
-  } as const,
-  {
-    id: 'zest',
-    url: 'https://app.zestprotocol.com',
-    logo: <ZestLogo size="32px" />,
-    title: 'Zest',
-    description:
-      'Commit sBTC to liquidity pools to earn fees and LP tokens in addition to basic sBTC rewards',
-    tvl: '950 BTC',
-    tvlUsd: '$57,950,000',
-    minCommitment: '0.007 BTC',
-    minCommitmentUsd: '$423.50',
-    apr: '5.3%',
-    payoutToken: 'sBTC',
-  } as const,
-] satisfies RewardProtocolInfo[];
+const logoMap: Record<string, ReactElement> = {
+  AlexLogo: <AlexLogo size="32px" />,
+  BitflowLogo: <BitflowLogo size="32px" />,
+  VelarLogo: <VelarLogo size="32px" />,
+  ZestLogo: <ZestLogo size="32px" />,
+};
```

Comment:
These should either be components `() => <AlexLogo />` or named lower case as jsx blocks